### ISSUE TITLE
It's not compatible with .net 8.0

### ISSUE THREAD

**Issue by contact2vikas** [rdvojmoc/DinkToPdf#188]
It's not compatible with .net 8.0

**Comment by BhangeeF16** [comment]
it is compatible with .net 8.0 I tested on a console application,

it throws #119 but it can be fixed using [@distantcam reply](https://github.com/rdvojmoc/DinkToPdf/issues/119#issuecomment-869296618)

**Comment by dqmarkmagaoay** [comment]
I agree with @BhangeeF16, we are using .net core 8 in our web API which uses this library and it works in azure app
